public/server/db.json

```
@@ -255,7 +255,15 @@
     "definition": "More Good Life"
   },
   "foss": {
-    "definition":"Free Open-Source Software"
+    "definition": "Free Open-Source Software"
+  },
+  "wagmi": {
+    "definition": "We're all gonna make it"
```

The first letter of each word should be in uppercase.
```suggestion
    "definition": "We're All Gonna Make It"
```